tests/foreman/api/test_contentmanagement.py

```diff
@@ -88,6 +89,8 @@ def test_positive_sync_repos_with_large_errata(self):
 
         :CaseLevel: Integration
 
+        :CaseImportance: High
```

Comment:
We have it at https://github.com/SatelliteQE/robottelo/blob/master/tests/foreman/api/test_contentmanagement.py#L13. I guess no need to add it for every test then.